client/src/components/aggregations/utils.js

```diff
@@ -216,8 +216,34 @@ export function reportsToEvents(reports) {
       url: Report.pathFor(r),
       classNames: [`event-${Report.getStateForClassName(r)}`],
       extendedProps: { ...r },
-      allDay:
-        !Settings.engagementsIncludeTimeAndDuration || r.duration === null
+      allDay: !Settings.engagementsIncludeTimeAndDuration || r.duration === null
     }
   })
 }
+export const COLOR_NAMES_TO_RGB = {
+  red: "rgb(155, 0, 0, ",
```

Comment:
could probably make use of https://github.com/d3/d3-color
and possibly create continuous color palettes/legends:
https://github.com/d3/d3-scale-chromatic/blob/master/README.md